free-2.md

```diff
@@ -1,44 +1,57 @@
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Lots of people like wine and food and consider themselves to be foodies. However, lots of people probably don't know be heart exactly which foods and wines go well togther, and want to improve their eating and drinking experiences. I could imagine a person who is eating a steak with potatoes, and wants to pair a wine with it but isn't sure which wine they want; perhaps they could even input to the program a list of wines that they are choosing between and the program/language could then help them choose the most appropriate choice. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL seems like a good choice because wine pairing is a fairly narrow domain, and users could use the language in order to educate themselves without having to go through too much other information. A language devoted strictly to this topic could be really helpful in assisting users expand their tastes for wine and food.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+My whole family are huge wine snobs, yet I find that I am often lacking in the taste refinement that they have. I feel like I would totally use an app like this to improve my own experiences with food and wine, and I feel like even experienced wine drinkers could learn some new things about the way they experience wine.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Pairing foods with appropriate wines
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I think there are two ways one could go about this. One option is that the user inputs specific foods or recipes - spaghetti marinara, grilled fish with soy sauce, etc - and then the program would provide either wine categories or specific wines that would work well with that food/recipe. This could be a tuples of wines/wine categories, and the amount between 0 and 1 that that wine is a good match. As such, programming in that language would look something like searching through a list of foods and then calling a findPairings on that food. This would be nice because it would be easiest for the user to get exactly the desired result.
+
+Another option (probably the easier to program) would be to have the user input flavors or ingredients that are dominant in the food they are eating, and then have those inputs evaluated in order to produce the best matching wines. This seems like what would be happening under the surface anyway in the first case, and would require more work from the user. This would possibly be more versatile, however, if the user wants to pair with a food that isn't well known or perhaps doesn't even have a well known name.
```

Comment:
Seems reasonable. Again, I am wondering how you are going build up this information database (i.e. where is ground truth?)